main4.js

```diff
@@ -0,0 +1,71 @@
+// 4a -- maak een functie die een string teruggeeft
+console.log('OPDRACHT 4a');
+
+function getTvObject(item){
+    let tvObject = item.brand + ' ' + item.type + ' - ' + item.name;
+    return tvObject;
+  }
+getTvObject(inventory[1]);
+
+// 4b -- maak een functie die de prijs format
+console.log('OPDRACHT 4b');
+
+function getTvPrice(item){
+    const price = '€' + item.price + ',-'
+    return price;
+}
+getTvPrice(inventory[1]);
+
+// 4c --
+console.log('OPDRACHT 4c');
+
+// from inch to cm
+function inchToCm(inch){
+    const cm = inch * 2.54;
+    return Math.floor(cm);
+}
+
+// functie uitschrijven
+function getTvRes(item){
+     let res = '';
+     let resArr = [];
+
+     for (let i = 0; i < item.length; i++){
+         res = item[i] + ' inch (' + inchToCm( item[i]) + ' cm)';
+         resArr.push(res);
+         res = resArr.join(" | ");
```

Comment:
Deze zou ik buiten de for-loop doen, omdat je dit resultaat enkel pas ziet als de loop gestopt is. Daarom kan die erbuiten en vanwege performance zou dat dan ook moeten.